[2021-03-30 20:43:22,418] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: CMAPS_SNOWFLAKE_INGESTION.DataExtraction 2021-03-31T00:02:59+00:00 [queued]>
[2021-03-30 20:43:22,425] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: CMAPS_SNOWFLAKE_INGESTION.DataExtraction 2021-03-31T00:02:59+00:00 [queued]>
[2021-03-30 20:43:22,425] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-30 20:43:22,425] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-30 20:43:22,425] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-30 20:43:22,438] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): DataExtraction> on 2021-03-31T00:02:59+00:00
[2021-03-30 20:43:22,441] {standard_task_runner.py:52} INFO - Started process 23017 to run task
[2021-03-30 20:43:22,450] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'CMAPS_SNOWFLAKE_INGESTION', 'DataExtraction', '2021-03-31T00:02:59+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/cmaps_pipeline.py', '--cfg-path', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmplhqw3xdj', '--error-file', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmpsbl9vkjv']
[2021-03-30 20:43:22,452] {standard_task_runner.py:77} INFO - Job 14: Subtask DataExtraction
[2021-03-30 20:43:22,497] {logging_mixin.py:104} INFO - Running <TaskInstance: CMAPS_SNOWFLAKE_INGESTION.DataExtraction 2021-03-31T00:02:59+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2021-03-30 20:43:22,526] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=CMAPS_SNOWFLAKE_INGESTION
AIRFLOW_CTX_TASK_ID=DataExtraction
AIRFLOW_CTX_EXECUTION_DATE=2021-03-31T00:02:59+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-31T00:02:59+00:00
[2021-03-30 20:43:22,538] {logging_mixin.py:104} WARNING - /Users/harika/Desktop/CSYE_7245/Assignment3/FastAPI/airflow/dags/cmaps_pipeline.py:23 ParserWarning: Falling back to the 'python' engine because the 'c' engine does not support regex separators (separators > 1 char and different from '\s+' are interpreted as regex); you can avoid this warning by specifying engine='python'.
[2021-03-30 20:43:37,593] {local_task_job.py:187} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-03-30 20:43:37,614] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 23017
[2021-03-30 20:43:37,615] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-03-30 20:43:37,726] {process_utils.py:66} INFO - Process psutil.Process(pid=23017, status='terminated', exitcode=0, started='20:43:22') (23017) terminated with exit code 0
